db/types.go

```diff
@@ -29,7 +29,7 @@ var (
 // DBConnection represents a connection to a versioned database.
 // Records are accessed via transaction objects, and must be safe for concurrent creation
 // and read and write access.
-// Past versions are only accessible read-only.
+// Past versions are only accessible read-only. TODO: do we want write access?
```

Comment:
If we're introducing TODOs, can we please reference a GitHub issue in the comment?

Reply:
I can remove it, this is more of a possible question for a later date, which will arise of its own accord if it's ever relevant.